libr/core/cmd_print.c

```diff
@@ -2431,10 +2434,15 @@ r_cons_pop();
 			if (show_comments) {
 				char *comment = r_core_anal_get_comments (core, addr);
 				if (comment) {
-					if (show_offset) {
-						r_cons_printf ("%s0x%08"PFMT64x" ", use_color? pal->offset: "", addr);
+					if (strcmp(comment, switchcmp)) {
```

Comment:
switchcmp may be null when not set, so check for null in here to avoid a crash

Reply:
This is what I was looking for. Thank you! :p